packages/client/components/GitLabScopingSearchCurrentFilters.tsx

```diff
@@ -0,0 +1,67 @@
+import styled from '@emotion/styled'
+import graphql from 'babel-plugin-relay/macro'
+import React from 'react'
+import {useFragment} from 'react-relay'
+import {PALETTE} from '../styles/paletteV3'
+import {GitLabScopingSearchCurrentFilters_meeting$key} from '../__generated__/GitLabScopingSearchCurrentFilters_meeting.graphql'
+
+const Wrapper = styled('div')({
+  width: '100%',
+  display: 'flex',
+  paddingLeft: '72px',
+  paddingTop: '8px'
+})
+
+const Description = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 16,
+  fontWeight: 500,
+  whiteSpace: 'nowrap'
+})
+
+const Items = styled('div')({
+  color: PALETTE.SLATE_600,
+  fontSize: 16,
+  fontWeight: 600,
+  fontStyle: 'italic',
+  padding: '0px 24px 0px 4px',
+  width: '100%',
+  whiteSpace: 'nowrap',
+  overflow: 'hidden',
+  textOverflow: 'ellipsis'
```

Comment:
I went with truncating & using an ellipsis if the "Current filters" text is too long. We discussed in the issue showing three filters and showing "and x more" if there's no room to show all filters, but three filters could take up half a line or several lines. This approach is also easy to implement and requires less code.